.travis.yml

```diff
@@ -60,8 +60,8 @@ install:
     - mysql zk_test < tests/test_dbs/core143.sql
     # run the upgrade from the 143 db
     - cp .env .env.local # start with fresh copy of .env.local
-    - 'sed -i -E "s/ZIKULA_INSTALLED=''(.*)''/ZIKULA_INSTALLED=''1.4.3''/" .env.local'
     - cp tests/services_custom.yaml config/services_custom.yaml
```

Comment:
Is `- 'sed -i -E "s/core_installed_version:(.*)/core_installed_version: ''1.4.3''/" config/services_custom.yaml'` missing here?